test/integration.js

```diff
@@ -15,12 +15,12 @@ describe('c8', () => {
       cwd: process.cwd()
     })
     output.toString('utf8').should.include(`
-------------|----------|----------|----------|----------|----------------|
-File        |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
-------------|----------|----------|----------|----------|----------------|
-All files   |    91.67 |      100 |       80 |    91.67 |                |
- normal.js  |    85.71 |      100 |       50 |    85.71 |       14,15,16 |
- timeout.js |      100 |      100 |      100 |      100 |                |
-------------|----------|----------|----------|----------|----------------|`)
```

Comment:
Are these changes expected?

Reply:
I had these results even without updating the dependencies, so I assumed that the change was caused by simply having a more recent version of Node.

By looking into it deeper, you are right that it reveals an issue. The functions in `fixtures/normal` are counted as `0/1` instead of `1/2`. The `apple` function is not detected as a function.

I should probably revert the change for this file: the tests were already failing before updating the dependencies and there's no CI anyway.